src/utilities/hooks/ModalPaymentContext/types.ts

```diff
@@ -3,16 +3,20 @@
  *
  * @typedef {Object} DefaultModalPaymentState
  * @property {number} amount - amount (in cents)
+ * @property {number} feesAmount - amount of fees (in cents)
+ * @property {FeeParams[]} fees - fee params used to calculate feesAmount
  * @property {boolean} customInput - if a user used the custom input
  * @property {boolean} close - modal for closing the payment modal
  * @property {number} modalView - expiration year
  *
  */
 
-import { BrowsePageSeller } from '../../api/types';
+import { BrowsePageSeller, FeeParams } from '../../api/types';
 
 export type ModalPaymentState = {
-  amount: string;
+  amount: string; // @TODO: refactor amount to be type number
+  feesAmount?: number;
```

Comment:
How come we need this separate amount and can't just sum fees?

Reply:
Ah, yeah, I thought about doing that, but decided to keep them separate as an easier way to create a line item for each during the CardDetailsModal step. (One line item for donation amount, second line item for fees).

Another potential way we can approach this is to maybe keep line items (instead of amount/feeAmount) at the context-level (acting like a "cart" as the user advances through the modals) and then just straight up submit those line items to Square on the final step. But I didn't explore that route too much, tbh!

Reply:
Yeah it would be nice to have a class of sorts that takes line items and has a calculated attributes of subtotal, fee total, etc.

Reply:
Yeah, agreed. And line items should probably be kept higher up, at like the modal payment context-level. Would be good to do in a refactor imo.